stdlib/public/Darwin/Foundation/Foundation.swift

```diff
@@ -29,7 +29,153 @@ public let NSNotFound: Int = .max
+///     """)
+///
+///     NSLocalizedString("book-button-title", value: "Book", comment: """
+///     verb: Title of the button that makes a reservation.
+///     """)
+///
+/// Use Only String Literals
+/// ------------------------
+///
+/// String literal values must be used with `key`, `tableName`, `value`, and
+/// `comment`.
+///
+/// Xcode does not evaluate interpolated strings and string variables when
+/// generating strings files from code. Using those language features will
+/// result in exported localizations that resemble the code expression instead
+/// of its value. Translators would then translate that exported value—leaving
+/// international users with a localized string containing code.
+///
+///     // Translators will see "1 + 1 = (1 + 1)".
+///     // International users will see a localization "1 + 1 = (1 + 1)".
+///     let localizedString = NSLocalizedString("string-interpolation",
+///                                             value: "1 + 1 = \(1 + 1)"
+///                                             comment: "A math equation.")
+///
+/// To dynamically insert values within localized strings, set `value` to a
+/// format string, and use `String.localizedStringWithFormat(_:_:)` to insert
+/// those values.
+///
+///     // Translators will see "1 + 1 = %d" (they know what "%d" means).
+///     // International users will see a localization of "1 + 1 = 2".
+///     let format = NSLocalizedString("string-literal",
+///                                    value: "1 + 1 = %d",
+///                                    comment: "A math equation.")
+///     let localizedString = String.localizedStringWithFormat(format, (1 + 1))
+///
+/// Multi-line string literals are technically supported, but will result in
+/// unexpected behavior during internationalization. A newline will be inserted
+/// before and after the body of text within the string, and translators will
+/// likely preserve those in their internaliations.
+///
+/// To preserve some of the aesthetics of having newlines in the string mirrored
+/// in their code representation, string literal concatenation with the `+`
+/// operator can be used.
+///
+///     NSLocalizedString("multi-line-string-literal",
+///                       value: """
+///     This multi-line string literal won't work as expected.
+///     An extra newline added to the beginning and end of the string.
+///     """,
+///                       comment: "The description of a sample of code.")
+///
+///     NSLocalizedString("string-literal-contatenation",
+///                       value: "This string literal concatenated with"
+///                            + "this other string literal works just fine.",
+///                       comment: "The description of a sample of code.")
+///
+/// Since comments aren't localized, multi-line string literals can be safely
+/// used with `comment`.
+///
+/// Restrict Autosynthesized Localizations
```

Comment:
Maybe "Work With Manually Managed Strings"?

Reply:
"Mitigate Mayhem With Manually Managed Strings " 😆

"Work With Manually Managed Strings" sounds much better. I'll go with that one. Thanks!